{ 'stand_up_back_200': { 'head': [],
                         'left_arm': [ { 'duration': 2.02,
                                         'name': u'stand_up_back_400',
                                         'positions': [ -2.3053,
                                                        1.2728,
                                                        -0.0044,
                                                        -0.0272,
                                                        -0.0055,
                                                        -1.5679,
                                                        -0.0004],
                                         'starttime': 0.0},
                                       { 'duration': 0.979,
                                         'name': u'stand_up_back_500',
                                         'positions': [ -2.3392,
                                                        1.6062,
                                                        0.0034,
                                                        0.0025,
                                                        -0.011,
                                                        -1.0039,
                                                        0.0003],
                                         'starttime': 2.02},
                                       { 'duration': 1.0,
                                         'name': u'stand_up_back_600',
                                         'positions': [ -2.0607,
                                                        1.8153,
                                                        0.0016,
                                                        0.0012,
                                                        -1.7053,
                                                        -1.0424,
                                                        -0.0029],
                                         'starttime': 2.999},
                                       { 'duration': 2.008,
                                         'name': u'stand_up_back_700',
                                         'positions': [ -0.9912,
                                                        1.8015,
                                                        -0.0085,
                                                        -0.015,
                                                        -1.6617,
                                                        -0.9969,
                                                        -0.0085],
                                         'starttime': 3.999},
                                       { 'duration': 1.987,
                                         'name': u'stand_up_back_800',
                                         'positions': [ -0.9867,
                                                        1.9962,
                                                        -0.014,
                                                        -0.0508,
                                                        -1.6029,
                                                        -1.0173,
                                                        -0.0121],
                                         'starttime': 6.007},
                                       { 'duration': 1.0,
                                         'name': u'stand_up_back_900',
                                         'positions': [ -0.5138,
                                                        1.803,
                                                        -0.0445,
                                                        -0.127,
                                                        -1.5892,
                                                        -1.0313,
                                                        -0.0202],
                                         'starttime': 7.994}],
                         'left_leg': [ { 'duration': 2.02,
                                         'name': u'stand_up_back_400',
                                         'positions': [ -0.0008,
                                                        -0.0008,
                                                        -0.9024,
                                                        2.8043,
                                                        0.5012,
                                                        0.0016],
                                         'starttime': 0.0},
                                       { 'duration': 0.933,
                                         'name': u'stand_up_back_500',
                                         'positions': [ -0.0012,
                                                        0.0065,
                                                        -0.4258,
                                                        2.8023,
                                                        0.9086,
                                                        0.0006],
                                         'starttime': 2.052},
                                       { 'duration': 1.0,
                                         'name': u'stand_up_back_600',
                                         'positions': [ -0.0074,
                                                        -0.016,
                                                        -0.0042,
                                                        2.833,
                                                        1.3035,
                                                        -0.0054],
                                         'starttime': 2.9850000000000003},
                                       { 'duration': 2.012,
                                         'name': u'stand_up_back_700',
                                         'positions': [ 0.0055,
                                                        0.0024,
                                                        0.3036,
                                                        2.6297,
                                                        1.688,
                                                        -0.0019],
                                         'starttime': 3.9850000000000003},
                                       { 'duration': 2.008,
                                         'name': u'stand_up_back_800',
                                         'positions': [ -0.0081,
                                                        0.002,
                                                        0.7479,
                                                        2.3325,
                                                        1.6868,
                                                        -0.0003],
                                         'starttime': 5.997},
                                       { 'duration': 1.0,
                                         'name': u'stand_up_back_900',
                                         'positions': [ 0.0054,
                                                        -0.0012,
                                                        1.1484,
                                                        2.024,
                                                        1.787,
                                                        -0.0003],
                                         'starttime': 8.004999999999999}],
                         'right_arm': [ { 'duration': 2.02,
                                          'name': u'stand_up_back_400',
                                          'positions': [ 2.3053,
                                                         -1.2728,
                                                         0.0044,
                                                         0.0272,
                                                         0.0055,
                                                         1.5679,
                                                         0.0004],
                                          'starttime': 0.0},
                                        { 'duration': 0.998,
                                          'name': u'stand_up_back_500',
                                          'positions': [ 2.3392,
                                                         -1.6062,
                                                         -0.0034,
                                                         -0.0025,
                                                         0.011,
                                                         1.0039,
                                                         -0.0003],
                                          'starttime': 2.02},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_back_600',
                                          'positions': [ 2.0607,
                                                         -1.8153,
                                                         -0.0016,
                                                         -0.0012,
                                                         1.7053,
                                                         1.0424,
                                                         0.0029],
                                          'starttime': 3.018},
                                        { 'duration': 1.98,
                                          'name': u'stand_up_back_700',
                                          'positions': [ 0.9912,
                                                         -1.8015,
                                                         0.0085,
                                                         0.015,
                                                         1.6617,
                                                         0.9969,
                                                         0.0085],
                                          'starttime': 4.018},
                                        { 'duration': 1.988,
                                          'name': u'stand_up_back_800',
                                          'positions': [ 0.9867,
                                                         -1.9962,
                                                         0.014,
                                                         0.0508,
                                                         1.6029,
                                                         1.0173,
                                                         0.0121],
                                          'starttime': 5.997999999999999},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_back_900',
                                          'positions': [ 0.5138,
                                                         -1.803,
                                                         0.0445,
                                                         0.127,
                                                         1.5892,
                                                         1.0313,
                                                         0.0202],
                                          'starttime': 7.986}],
                         'right_leg': [ { 'duration': 2.02,
                                          'name': u'stand_up_back_400',
                                          'positions': [ 0.0008,
                                                         0.0008,
                                                         0.9024,
                                                         -2.8043,
                                                         -0.5012,
                                                         -0.0016],
                                          'starttime': 0.0},
                                        { 'duration': 0.989,
                                          'name': u'stand_up_back_500',
                                          'positions': [ 0.0012,
                                                         -0.0065,
                                                         0.4258,
                                                         -2.8023,
                                                         -0.9086,
                                                         -0.0006],
                                          'starttime': 2.02},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_back_600',
                                          'positions': [ 0.0074,
                                                         0.016,
                                                         0.0042,
                                                         -2.833,
                                                         -1.3035,
                                                         0.0054],
                                          'starttime': 3.009},
                                        { 'duration': 1.986,
                                          'name': u'stand_up_back_700',
                                          'positions': [ -0.0055,
                                                         -0.0024,
                                                         -0.3036,
                                                         -2.6297,
                                                         -1.688,
                                                         0.0019],
                                          'starttime': 4.009},
                                        { 'duration': 2.008,
                                          'name': u'stand_up_back_800',
                                          'positions': [ 0.0081,
                                                         -0.002,
                                                         -0.7479,
                                                         -2.3325,
                                                         -1.6868,
                                                         0.0003],
                                          'starttime': 5.995},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_back_900',
                                          'positions': [ -0.0054,
                                                         0.0012,
                                                         -1.1484,
                                                         -2.024,
                                                         -1.787,
                                                         0.0003],
                                          'starttime': 8.003}],
                         'torso': []}}
